packages/onchainkit/src/wallet/components/WalletAdvancedSwap.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { PressableIcon } from '@/internal/components/PressableIcon';
 import { backArrowSvg } from '@/internal/svg/backArrowSvg';
-import { border, cn } from '@/styles/theme';
```

Comment:
seems random. linter?

Reply:
Yeah, needed lint to pass to be able to merge this